modules/security-context-constraints-requiring.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-requiring_{context}"]
+= Configuring a workload to require a specific SCC
+
+You can configure a workload to require a certain security context constraint (SCC). This is useful in scenarios where you want to pin to a specific SCC to the workload or if you want to prevent your required SCC from being preempted by another SCC in the cluster.
+
+To require a specific SCC, set the `openshift.io/required-scc` annotation on your workload. You can set this annotation on any resource that can set a pod manifest template, such as a deployment or daemon set.
+
+The SCC must already exist in the cluster and must be applicable to the workload, otherwise pod admission will fail.
+
+[WARNING]
+====
+Do not change the `openshift.io/required-scc` annotation in the live pod's manifest; the pod admission will fail.
+====
+
+.Prerequisites
+
+// TODO: Verify that there is no case where you have to be the cluster admin here
+
+* The SCC must already exist in the cluster.
+
+.Procedure
+
+. Create a YAML file for the deployment and set a required SCC by setting the `openshift.io/required-scc` annotation:
++
+.Example `deployment.yaml`
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Deployment
+apiVersion: apps/v1
+spec:
+# ...
+  template:
+    metadata:
+      annotations:
+        openshift.io/required-scc: "my-scc" <1>
+# ...
+----
+<1> Specify the name of the SCC to require.
+
+. Create the resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f deployment.yaml
+----
+
+.Verification
+
+* Verify that the deployment used the specified SCC.
+
+.. View the value of the pod's `openshift.io/scc` annotation by running the following command:
```

Comment:
@s-urbaniak @dpuniaredhat we've been discussing how to verify this in https://issues.redhat.com/browse/AUTH-403. I went with checking the annotation, for the reasons I stated in my latest comment. Let me know if that works, or if we need to rethink this.

Reply:
Looks Good to me :+1: